lib/common/options.c

```diff
@@ -283,7 +283,7 @@ static const pcmk__cluster_option_t cluster_options[] = {
          * is unset or fails to validate.
          */
         PCMK_OPT_STONITH_WATCHDOG_TIMEOUT, NULL, PCMK_VALUE_TIMEOUT, NULL,
```

Comment:
similarly, use PCMK_VALUE_DURATION here, then the description doesn't need to mention negative values